# mcl_2d:
#   ros__parameters:
#     param_name: UAAAAAAAAAAAAAAAAAAAAAAAAAAA!!
#     # 他のパラメータをここに追加

/**:
  ros__parameters:
    global_frame_id: "map"
    footprint_frame_id: "base_footprint"
    odom_frame_id: "odom"
    base_frame_id : "base_link"
    odom_freq: 10
